pkg/kubelet/qos/util/qos.go

```diff
@@ -19,12 +19,7 @@ package util
 import (
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/resource"
-)
-
-const (
-	Guaranteed = "Guaranteed"
-	Burstable  = "Burstable"
-	BestEffort = "BestEffort"
+	qostypes "k8s.io/kubernetes/pkg/kubelet/qos/types"
```

Comment:
Prefer we just have a /qos/types.go. Is there some import dependency that is a problem?  If so I would prefer to get rid of qos/util and just have these in qos/helpers.go